app/components/candidate_interface/restructured_work_history_review_component.html.erb

```diff
@@ -0,0 +1,26 @@
+<section>
+  <% @work_history_with_breaks.each do |entry| %>
+    <% if entry.is_a?(ApplicationWorkExperience) %>
+
+      <%= render(CandidateInterface::RestructuredWorkHistoryWorkExperienceComponent.new(work_experience: entry, editable: @editable)) %>
+
+    <% elsif @editable && entry.is_a?(WorkHistoryWithBreaks::BreakPlaceholder) %>
+
+      <%= render(CandidateInterface::RestructuredWorkHistoryBreakPlaceholderComponent.new(break_period: entry)) %>
+
+    <% elsif entry.is_a?(ApplicationWorkHistoryBreak) %>
+
+      <%= render(CandidateInterface::RestructuredWorkHistoryWorkBreakComponent.new(work_break: entry, editable: @editable)) %>
```

Comment:
I think all of these components can sit in a restructured_work_history folder

So they'd be:

```
CandidateInterface::RestructuredWorkHistory::WorkExperienceComponent
CandidateInterface::RestructuredWorkHistory::BreakPlaceholderComponent
CandidateInterface::RestructuredWorkHistory::WorkBreakComponent
```

We've got so many components at the moment in the candidate UI folder atm and it's hard to find things.

I'm going to add a tech debt card to try and group the components and put them in appropriate folders

Reply:
I think I'd be tempted to rename `RestructuredWorkHistoryBreakPlaceholderComponent` to something like `RestructuredWorkHistoryGapComponent`, as the gap could just as well be filled by a work break or a job?